packages/react-server-native-relay/src/__tests__/ReactFlightNativeRelay-test.internal.js

```diff
@@ -7,6 +7,8 @@
 
 'use strict';
 
+global.TextEncoder = require('util').TextEncoder;
```

Comment:
This doesn’t exist in FB runtimes so this wouldn’t work but also it shouldn’t be needed in this test since it shouldn’t be testing node here.

Reply:
Hmm odd, I confirmed the test fails without it. sign of an bigger issue?